<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>practiceWidget</class>
 <widget class="QWidget" name="practiceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>760</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>760</width>
    <height>760</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>760</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>130</y>
     <width>271</width>
     <height>531</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <column>
    <property name="text">
     <string>手顺</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>评分</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="blackLabel">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>30</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>黑方优势</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>20</y>
     <width>81</width>
     <height>701</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="whiteLabel">
   <property name="geometry">
    <rect>
     <x>767</x>
     <y>700</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>白方优势</string>
   </property>
  </widget>
  <widget class="QPushButton" name="superForwardBtn">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>680</y>
     <width>41</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="superBackwardBtn">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>680</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="playBtn">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>680</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>|&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backwardBtn">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>680</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="forwardBtn">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>680</y>
     <width>41</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openBtn">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>70</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:15px;
padding : 0 4px;
background: #FFC0CB;
border -radius:10px;
color: rgb(255, 255, 255);
font: 24px
}
QPushButton:hover:!pressed

{
background:#FFC0CB;
}
QToolButton
{

border-radius:10px;
bacground:red;
color:red;
}

QPushButton:pressed
{
              font-size: 19px;
              transition: 0.9s; 
}</string>
   </property>
   <property name="text">
    <string>导入棋谱</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveBtn">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>20</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:15px;
padding : 0 4px;
background: #FFC0CB;
border -radius:10px;
color: rgb(255, 255, 255);
font: 24px
}
QPushButton:hover:!pressed

{
background:#FFC0CB;
}
QToolButton
{

border-radius:10px;
bacground:red;
color:red;
}

QPushButton:pressed
{
              font-size: 19px;
              transition: 0.9s; 
}</string>
   </property>
   <property name="text">
    <string>保存棋谱</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startBtn">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>70</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:15px;
padding : 0 4px;
background: #FFC0CB;
border -radius:10px;
color: rgb(255, 255, 255);
font: 24px
}
QPushButton:hover:!pressed

{
background:#FFC0CB;
}
QToolButton
{

border-radius:10px;
bacground:red;
color:red;
}

QPushButton:pressed
{
              font-size: 19px;
              transition: 0.9s; 
}</string>
   </property>
   <property name="text">
    <string>重新开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="returnBtn">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>20</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:15px;
padding : 0 4px;
background: #FFC0CB;
border -radius:10px;
color: rgb(255, 255, 255);
font: 24px
}
QPushButton:hover:!pressed

{
background:#FFC0CB;
}
QToolButton
{

border-radius:10px;
bacground:red;
color:red;
}

QPushButton:pressed
{
              font-size: 19px;
              transition: 0.9s; 
}</string>
   </property>
   <property name="text">
    <string>返回菜单</string>
   </property>
  </widget>
  <zorder>tableWidget</zorder>
  <zorder>progressBar</zorder>
  <zorder>whiteLabel</zorder>
  <zorder>blackLabel</zorder>
  <zorder>superForwardBtn</zorder>
  <zorder>superBackwardBtn</zorder>
  <zorder>playBtn</zorder>
  <zorder>backwardBtn</zorder>
  <zorder>forwardBtn</zorder>
  <zorder>openBtn</zorder>
  <zorder>saveBtn</zorder>
  <zorder>startBtn</zorder>
  <zorder>returnBtn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
